test/src/integration/integration_test.go

```diff
@@ -159,20 +164,42 @@ func runAnsiblePlaybooks(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	err = ioutil.WriteFile("./inventory.ini", []byte(lookupTargetValue(t, "kubernetes", "inventory_ini")), 0644)
+	err = ioutil.WriteFile("./inventories/kubernetes", []byte(lookupTargetValue(t, "kubernetes", "inventory_ini")), 0644)
 	if err != nil {
 		t.Fatal(err)
 	}
 
 	// Install tools
-	runAnsiblePlaybook(t, k8sModuleDir, []string{"./playbooks/playbook-install-tools.yml", "-e", "base_local_directory=../../../../", "-e", "install_awscli=" + installAwscli})
+	runAnsiblePlaybook(t, k8sModuleDir, "../inventories", []string{"./playbooks/playbook-install-tools.yml", "-e", "base_local_directory=../../../../", "-e", "install_awscli=" + installAwscli})
```

Comment:
Just a question.
`inventories` directory is the parent of the current directory?
Not sure but it looks like the current directory.
https://github.com/scalar-labs/scalar-terratest/pull/12/files#diff-70f32499abec27d850272b508cf4a5da9c4c01ebb45970a2f619787d6b42bd6bR167